test-runner/test_module_send_telemetry.py

```diff
@@ -40,3 +42,37 @@ def test_module_send_event_to_iothub():
     module_client.disconnect()
     log_message("disconnecting eventhub client")
     eventhub_client.disconnect()
+
+
+@pytest.mark.asyncio
+@pytest.mark.testgroup_edgehub_module_client
+@pytest.mark.testgroup_iothub_module_client
+@pytest.mark.callsSendEvent
+async def test_module_send_event_to_iothub_async():
+
+    executor = concurrent.futures.ThreadPoolExecutor()
+    loop = asyncio.get_event_loop()
+
+    log_message("connecting module client")
+    module_client = await connections.connect_test_module_client_async()
+    log_message("connecting eventhub client")
+    eventhub_client = await loop.run_in_executor(executor, connections.connect_eventhub_client)
```

Comment:
Why is this being run in a ThreadPool anyway? Couldn't you just call the function?